source/multigrid/mg_transfer_component.cc

```diff
@@ -531,16 +531,16 @@ MGTransferComponentBase::build(const DoFHandler<dim, spacedim> &mg_dof)
                   for (types::global_dof_index i = 0; i < n_dofs; ++i)
                     {
                       SparseMatrix<double>::iterator
-                        anfang = prolongation_matrices[level]
```

Comment:
I personally think, we should use more german!

Reply:
I could be convinced to leave it :)

Reply:
Ha, that's a good one :-)

Reply:
I bet that dates back to 1999 when @guidokanschat wrote these lines!